### ISSUE TITLE
Re-order elements in activated "Leave a Reply" box

### ISSUE THREAD

**Issue by dgroos** [LeGroup/Fle4#56]
Upon clicking a "Reply" button in either thread or map view a very simply dialog box is presented. However, upon selecting the knowledge type, the information on the selected KB appears immediately below the new KT, pushing the rest of the box (the comment field, the main-idea field and the post-comment button) to the bottom and often out of site off the display. This is a bit unsettling. 

Instead, upon selecting the KT, have its corresponding information/description appear at the bottom of the page, UNDER the comment field, main-idea field and the post-comment button.

**Comment by tarmot** [comment]
Well, this is a design decision, forcing people to read (or intentionally skip) the instructions. If they are moved to the end, they will be skipped more easily.

**Comment by dgroos** [comment]
True, then it becomes a question as to what provides better support for the users: 
--KT information in the middle of the space prompting students to read/reread it
--KT info at end so comment and reply are visually coupled--no 'distracting' info between them.
My guess is that different users have different needs and indeed that the needs of any user change over time as familiarity with the KB process grows. I'll do a simple user-opinion survey and get stu. feedback on this.

**Comment by tarmot** [comment]
Originally (Fle3 era) the KT information was on one side. So you had the original text and your reply, and then on another column the KT info. With current responsive designs, two columns aren't really available...
